de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/mat/Exporter.java

```diff
@@ -0,0 +1,290 @@
+package de.dlr.sc.virsat.model.dvlm.mat;
+
+import org.eclipse.emf.common.util.EList;
+
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyBoolean;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyEReference;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyEnum;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyFloat;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyInt;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyResource;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyString;
+import de.dlr.sc.virsat.model.dvlm.categories.ATypeInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.CategoryAssignment;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.BooleanProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.EReferenceProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.EnumProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.FloatProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.IntProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.ReferenceProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.ResourceProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.StringProperty;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.APropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ArrayInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ComposedPropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.EReferencePropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.EnumUnitPropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ReferencePropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ResourcePropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.UnitValuePropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ValuePropertyInstance;
+import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.util.PropertyinstancesSwitch;
+import de.dlr.sc.virsat.model.dvlm.structural.StructuralElementInstance;
+import us.hebi.matlab.mat.format.Mat5;
+import us.hebi.matlab.mat.types.Array;
+import us.hebi.matlab.mat.types.Cell;
+import us.hebi.matlab.mat.types.MatFile;
+import us.hebi.matlab.mat.types.Struct;
+
+/**
+ * Class for exporting data to .mat
+ */
+public class Exporter {
+
+	/**
+	 * creates a new .mat with everything inside the given sei
+	 * @param seiRoot sei to export
+	 */
+	public MatFile exportSei(StructuralElementInstance seiRoot) {
+		MatFile matfile = Mat5.newMatFile();		
+		Struct struct = Mat5.newStruct();
+		struct.set("type", Mat5.newString(seiRoot.getType().getName()))

```

Comment:
I think we should consider making a class holding the keys like "type" or "uuid" as final static strings.
This would make sure that is easy to change them (for example if we want to change lower case to upper case or something like that) and it would probably usueful later in the importer since we will need the keys again there.

Reply:
@SaMuellerDLR  
I changed it, but I am not sure if it is the way you wanted.